sdks/cpp/common/include/IParam.h

```diff
@@ -34,11 +34,14 @@
 #include <interface/param.pb.h>
 
 namespace catena {
-class Value; // forward reference
-class Param; // forward reference
-
 namespace common { 
 
+/**
+ * @brief EmptyValue is used for commands that do not have a value
+ */
+struct EmptyValue {};
+static EmptyValue emptyValue;
```

Comment:
this will cause emptyValue to be defined in every .cpp file that includes this header. better to do this:

IParam.h
``` .cpp
struct EmptyValue {};
extern EmptyValue emptyValue;
```

IParam.h:
``` .cpp
#include <IParam.h>
EmptyValue emptyValue;
```

```
```